airbyte-server/src/main/java/io/airbyte/server/handlers/SchedulerHandler.java

```diff
@@ -256,10 +253,9 @@ public SourceDefinitionSpecificationRead getSourceDefinitionSpecification(final
       throws ConfigNotFoundException, IOException, JsonValidationException {
     final UUID sourceDefinitionId = sourceDefinitionIdRequestBody.getSourceDefinitionId();
     final StandardSourceDefinition source = configRepository.getStandardSourceDefinition(sourceDefinitionId);
-    final SynchronousResponse<ConnectorSpecification> response = specFetcher.getSpecJobResponse(source);
-    final ConnectorSpecification spec = response.getOutput();
+    final ConnectorSpecification spec = source.getSpec();
     final SourceDefinitionSpecificationRead specRead = new SourceDefinitionSpecificationRead()
-        .jobInfo(jobConverter.getSynchronousJobRead(response))
+        .jobInfo(jobConverter.getSynchronousJobRead(SynchronousJobMetadata.mock(ConfigType.GET_SPEC)))
```

Comment:
JobInfo is a required field on Source/DestinationSpecificationRead, so I chose to add this mocked job info rather than trying to remove it from that struct, as I'm not completely sure what dependencies the UI has on this field